log/log.go

```diff
@@ -62,7 +62,7 @@ const auditTag = "[AUDIT]"
 // and also writes to stdout/stderr. It is safe for concurrent use.
 func New(log *syslog.Writer, stdoutLogLevel int, syslogLogLevel int) (Logger, error) {
 	if log == nil {
-		return nil, errors.New("Attempted to use a nil System Logger.")
+		return nil, errors.New("Attempted to use a nil System Logger")
```

Comment:
Is the linter not complaining about the capital letters at the start of the message too? Those are definitely frowned upon, I'm surprised it didn't say anything

Reply:
I'm going off of the squiggly lines in vscode complaining about the punctuation at the end of the sentence. Deleting the period cleaned up the squiggly line so 🤷🏼 .